osm2streets/src/render.rs

```diff
@@ -491,3 +501,56 @@ fn make_sidewalk_corners(streets: &StreetNetwork, intersection: &Intersection) -
     }
     results
 }
+
+fn draw_stop_lines(
+    lane: &LaneSpec,
+    center: &PolyLine,
+    road: &Road,
+) -> Vec<(Polygon, &'static str)> {
+    let mut results = Vec::new();
+
+    if !matches!(
+        lane.lt,
+        LaneType::Driving | LaneType::Bus | LaneType::Biking
+    ) {
+        return results;
+    }
+    let thickness = Distance::meters(0.5);
+
+    let stop_line = if lane.dir == Direction::Fwd {
+        &road.stop_line_end
+    } else {
+        &road.stop_line_start
+    };
+
+    // The vehicle line
+    if let Some(dist) = stop_line.vehicle_distance {
+        if let Ok((pt, angle)) = center.dist_along(dist) {
+            results.push((
+                Line::must_new(
+                    pt.project_away(lane.width / 2.0, angle.rotate_degs(90.0)),
+                    pt.project_away(lane.width / 2.0, angle.rotate_degs(-90.0)),
+                )
+                .make_polygons(thickness),
+                "vehicle stop line",
+            ));
+        }
+    }
+
+    if let Some(dist) = stop_line.bike_distance {
+        if let Ok((pt, angle)) = center.dist_along(dist) {
+            results.push((
+                Line::must_new(
+                    pt.project_away(lane.width / 2.0, angle.rotate_degs(90.0)),
+                    pt.project_away(lane.width / 2.0, angle.rotate_degs(-90.0)),
+                )
+                .make_polygons(thickness),
+                "bike stop line",
+            ));
+        }
+    }
+
+    // TODO Change the rendering based on interruption too
```

Comment:
We can iterate here to show give way / stop / signal, and to put a bike symbol in the bike box, etc